client/my-sites/exporter/post-type-options.jsx

```diff
@@ -69,28 +74,39 @@ const PostTypeOptions = React.createClass( {
 	},
 
 	renderFields() {
-		const { options } = this.props;
+		const {
+			options,
+			siteId,
+			postType,
+			values,
+		} = this.props;
 
 		const Field = ( props ) => {
```

Comment:
Any reason why this needed to be defined in a closure? Might be out of scope for this PR though.